idl/github.com/uber/cadence/shared.thrift

```diff
@@ -120,6 +121,9 @@ enum EventType {
   ChildWorkflowExecutionCanceled,
   ChildWorkflowExecutionTimedOut,
   ChildWorkflowExecutionTerminated,
+  SignalExternalWorkflowExecutionInitiated,
+  SignalExternalWorkflowExecutionFailed,
+  ExternalWorkflowExecutionSignalRequested,
```

Comment:
This should be ExternalWorkflowExecutionSignaled

Reply:
Changed. Thanks for catching this!